Extension/src/common.ts

```diff
@@ -912,6 +912,26 @@ export async function promptReloadWindow(message: string): Promise<void> {
     }
 }
 
+export async function promptCDDFailed(copy?: boolean): Promise<void> {
+    const copyMessage: string = localize("cdd.copyMessage", "Failed to copy declaration or definition to clipboard.");
+    const createMessage: string = localize("cdd.createMessage", "Failed to create declaration or definition.");
+
+    if (copy) {
+        await vscode.window.showInformationMessage(copyMessage);
+    } else {
+        await vscode.window.showInformationMessage(createMessage);
+    }
+}
+
+export async function addTrustedCompiler(compilers: string[], path: string): Promise<string[]> {
```

Comment:
Is `addTrustedCompiler` part of this change?

You may need to rebase this feature branch.

Reply:
No, you should remove it -- it looks like it was moved to client.ts.